x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/evaluate_panel.tsx

```diff
@@ -139,35 +258,40 @@ export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus }) => {
               )}
             </span>
           </EuiTitle>
+          {generalizationDocsCount !== null && (
+            <EuiText size="xs" color="subdued">
+              <FormattedMessage
+                id="xpack.ml.dataframe.analytics.regressionExploration.generalizationDocsCount"
+                defaultMessage="{docsCount} docs evaluated"
```

Comment:
This should include syntax for one / plural cases, for when there is 1 doc.

Reply:
Added in 494ac87820e7fc2863e03dc244f44c9a290bff4c